Glassmorphism-creative-app/README.md

```diff
@@ -0,0 +1,7 @@
+# Glassmorphism Creative Cloud app design with HTML and CSS
+
+## Description
+
+Attempt to design Glassmorphisam app with HTML & CSS. Tailwind CSS was used for this project.
+
+Live preview: https://codepen.io/sujon-khan-tonmoy/pen/QWmgzVq
```

Comment:
The link should be wrapped with a greater than and a less than symbol. i.e., `<https://codepen.io/sujon-khan-tonmoy/pen/QWmgzVq`. Read the relevant guide [here](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md034) to understand why.